components/Books.js

```diff
@@ -268,6 +275,42 @@ const Books = () => {
             </select>
           </div>
           <div className="section">{renderContent(option)}</div>
+
```

Comment:
We should [https://www.mediawiki.org/wiki/API:Emailuser#Checking_emailable_status](check if the user has an email) and keep the checkbox disabled if the user doesn't have email and on hover show the message: "No email associated with this user account."